### DISCUSSION TITLE
Date Inserted is before we went live with EMu, how did that happen?

### DISCUSSION THREAD

**Discussion by fmjjones** [fieldmuseum/EMu-Documentation#1038] [FAQs] [answered]
When data that existed prior to EMu implementation had record insertion dates associated, the import process forced the existing data into the Adm Inserted by [AdmInsertedBy] and Insertion date [AdmDateInserted] fields.

**Comment by rondlg** [comment, marked as answer]
Example: esites IRN 34014
- AdmDateInserted:[2002  ]

@peteherbst "I already coded a fix to handle year only but just wanted to give you an example record, looks like it’s the AdmDateInserted that originally was the issue"
